tape/src/main/java/com/squareup/tape2/ObjectQueue.java

```diff
@@ -49,7 +49,7 @@ public boolean isEmpty() {
    * If the queue's {@link #size()} is less than {@code max} then only {@link #size()} entries
    * are read.
    */
-  public List<T> peek(int max) throws IOException {
+  public final List<T> peek(int max) throws IOException {
```

Comment:
I think both of these should stay non-final since it's possible to provide more efficient implementations in subclasses.